### PR TITLE
Fix receipts

### PR DESCRIPTION
**Pull request by iltoga**
## Description
Fix published receipts selection and validation logic

### CODE REVIEW

**Review by capt4ce — APPROVED** [review]

File: common/constant/smith.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ import (
 
 var (
 	CoinbaseTotalDistribution        int64   = 3000000 * OneZBC // 3 million * 10^8 in production
-	CoinbaseTime                     int64   = 5 * OneYear      // 5 years in production
+	CoinbaseTime                     int64   = OneYear          // 15 years in production
```
Comment (iltoga) [inline common/constant/smith.go:58]: is this one correct. in the comments says 15 years in prod

File: core/service/blockMainService.go [hunk 2]
```diff
@@ -1795,6 +1795,7 @@ func (bs *BlockService) ProcessQueueBlock(block *model.Block, peer *model.Peer)
 	if len(txRequiredByBlock) == 0 {
 		err := bs.ProcessCompletedBlock(block)
 		if err != nil {
+			bs.Logger.Error("ProcessQueueBlock") //STEF temp delete
```
Comment (iltoga) [inline core/service/blockMainService.go:1798]: this should be deleted I think

File: core/service/blockMainService.go [hunk 3]
```diff
@@ -1838,6 +1839,7 @@ func (bs *BlockService) ReceivedValidatedBlockTransactionsListener() observer.Li
 				for _, block := range completedBlocks {
 					err := bs.ProcessCompletedBlock(block)
 					if err != nil {
+						bs.Logger.Error("ReceivedValidatedBlockTransactionsListener") //STEF temp delete
```
Comment (iltoga) [inline core/service/blockMainService.go:1842]: this should be deleted I think

File: readme.md [hunk 4]
```diff
@@ -74,23 +74,7 @@ To make use of the `Makefile` please rename `github.token.example` to `github.to
...
-Flags:
-      --config-path string      Configuration path (default "./")
-      --config-postfix string   Configuration version
-      --debug                   Run on debug mode
-  -h, --help                    help for this command
-      --profiling               Run with profiling
-      --use-env                 Running node without configuration file
-
-Use " [command] --help" for more information about a command.
+./zoobc
```
Comment (iltoga) [inline readme.md:77]: this one seems wrong.. maybe we should keep the version from develop.. I am not sure though

**Pull request merged**